pixeltable/metadata/schema.py

```diff
@@ -303,3 +303,11 @@ def as_dict(self) -> dict[str, Any]:
             'table_version_md': dataclasses.asdict(self.version_md),
             'table_schema_version_md': dataclasses.asdict(self.schema_version_md),
         }
+
+    @classmethod
+    def from_dict(cls, data_dict: dict[str, Any]) -> 'FullTableMd':
+        return FullTableMd(
+            tbl_md=TableMd(**data_dict['table_md']),
```

Comment:
Use `md_from_dict` for these?

`tbl_md=md_from_dict(TableMd, data['table_md'])`

etc